Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
Could be a problema on my Xcode installation. Even 1.5.8 formula doesn't
build with the same error, my previous installation was done using a
"bottled" version.

Does this help?

% brew gist-logs unbound

https://gist.github.com/cfdbc58665104f9cb38cb61da7305f6a

On Mon, Jul 18, 2016 at 1:28 PM Dominyk Tiller <notifications@github.com>
wrote:

> In Formula/unbound.rb
> <https://github.com/Homebrew/homebrew-core/pull/3080#discussion_r71180492>
> :
>
> > @@ -13,11 +13,13 @@ class Unbound < Formula
> >
> >    depends_on "openssl"
> >    depends_on "libevent"
> > +  depends_on "expat"
>
> Do you have a gist-logs for that build at all, by any chance? It's
> entirely possible the OS X expat is too old now, just nice to check we're
> not adding dependencies unnecessarily.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/3080/files/bae0c807e72cb3fde2dff0a9fabb6c40486b6982#r71180492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATkHVWbg-jOxETpgoBseZBV8J25MzeYiks5qW6mdgaJpZM4JOknY>
> .
>